config/locales/modules/child-development-and-the-eyfs.yml

```diff
@@ -1103,18 +1103,8 @@ en:
           Supporting material is available on the [Foundation Years website](https://foundationyears.org.uk/).
 
           See [guidance to provide support](https://www.gov.uk/government/publications/progress-check-at-age-2) when completing the early years foundation stage (EYFS) progress check at age 2.
-
-      1-3-2:
-        heading: Reflect on your learning
-        body: |
-          To help DfE to measure our impact, please answer the following questions. 
-
-          Answers will be anonymous, and we will not know what you individually select.
-
-          To what extent do you agree with the following statements:
       
       1-3-2-6:
         heading: Thank you
         body: |
-          You can also <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-Qjm3BfYzDVIhkQadsLBtTxUQVY5VUpLRkI4T0owS0VPRk0zSkQzSjZHTiQlQCN0PWcu" target="blank">give feedback about the training (opens in new window). 
-    
+          You can also <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-Qjm3BfYzDVIhkQadsLBtTxUQVY5VUpLRkI4T0owS0VPRk0zSkQzSjZHTiQlQCN0PWcu" target="blank">give feedback about the training (opens in new window).
```

Comment:
We have this as an env/credential we should pass it in

Reply:
It has been pointed out to me that there's a different office form for each module, so the link is different each time